server/config/db.config.ts

```diff
@@ -0,0 +1,5 @@
+import { ConfigService } from "@nestjs/config";
```

Comment:
issue: do not use this file and remove. Each service/module/controller should handle their own call to the config service

Reply:
Resolution for this problem not founded yet, FIXME comment add on dbConfig file.